TRADEMARK-COMMITTEE.md

```diff
@@ -58,12 +58,12 @@ subject to conformance and third-party licensing.
 
 ## Committee Meetings
 
-The committee meets as-needed. Meetings are held online.
+The trademark committee currently meets weekly, or as-needed. Meetings are open to public and held online.
```

Comment:
Why not include a link to the meeting invite here?

Reply:
I'm guessing it's because then we need to remember to update the link here when it's updated on Google Calendar.